docker-compose.yml

```diff
@@ -159,6 +159,56 @@ services:
     restart: always
     volumes:
       - aerie_file_store:/usr/src/app/merlin_file_store:ro
+  aerie_scheduler_worker_1:
+    build:
+      context: ./scheduler-worker
+      dockerfile: Dockerfile
+    container_name: aerie_scheduler_worker_1
+    depends_on: ["postgres"]
+    environment:
+      MERLIN_GRAPHQL_URL: http://hasura:8080/v1/graphql
+      SCHEDULER_LOCAL_STORE: /usr/src/app/scheduler_file_store
+      SCHEDULER_WORKER_DB: "aerie_scheduler"
+      SCHEDULER_WORKER_DB_PASSWORD: "${AERIE_PASSWORD}"
+      SCHEDULER_WORKER_DB_PORT: 5432
+      SCHEDULER_WORKER_DB_SERVER: postgres
+      SCHEDULER_WORKER_DB_USER: "${AERIE_USERNAME}"
+      SCHEDULER_OUTPUT_MODE: UpdateInputPlanWithNewActivities
+      SCHEDULER_RULES_JAR: /usr/src/app/merlin_file_store/scheduler_rules.jar
+      JAVA_OPTS: >
+        -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005
+        -Dorg.slf4j.simpleLogger.defaultLogLevel=DEBUG
+        -Dorg.slf4j.simpleLogger.log.com.zaxxer.hikari=INFO
+        -Dorg.slf4j.simpleLogger.logFile=System.err
+    ports: ["5009:5005", "27189:8080"]
```

Comment:
deploy/docker compose says 27188

Reply:
Good catch. Since the deployment stack only has 1 merlin worker and 1 scheduler server theres a small gap in the sequential port numbering in `deployment`. When I wrote this I figured we'd not introduce a gap in the port numbering, but I think having this stack's ports consistent with our development stack makes things far less confusing so I've changed fixed this :)

`deployment/`'s port is now 27189 for the scheduler worker 1.